R/scmake.R

```diff
@@ -200,8 +200,9 @@ is_ind_file <- function(target_names, ind_ext=getOption("scipiper.ind_ext")) {
 #' as_ind_file('mydata.rds', ind_ext='st') # 'mydata.rds.st'
 #' as_ind_file('mydata.rds.ind') # Error: "data_file is an indicator file already"
 as_ind_file <- function(data_file, ind_ext=getOption("scipiper.ind_ext")) {
-  if(is_ind_file(data_file, ind_ext=ind_ext)) {
-    stop('data_file is an indicator file already')
+  are_ind_files <- is_ind_file(data_file, ind_ext=ind_ext)
+  if(any(are_ind_files)) {
+    stop(paste(data_file[are_ind_files], 'are/is already an indicator file(s)'))
```

Comment:
well...closer. but a comma-separated list would be better than passing a vector of messages to `stop`. testing what's above with dummy data, i see:
```
> data_file = c('data.ind','data','mydata.ind')
> are_ind_files <- is_ind_file(data_file)
> are_ind_files
[1]  TRUE FALSE  TRUE
> any(are_ind_files)
[1] TRUE
> stop(paste(data_file[are_ind_files], 'are/is already an indicator file(s)'))
Error: data.ind are/is already an indicator file(s)mydata.ind are/is already an indicator file(s)
```
whereas a more readable message would be:
`Error: already an indicator file: data.ind, mydata.ind`